### ISSUE TITLE
TreeList improvements.

### ISSUE THREAD

**Issue by MilenaCh** [telerik/kendo-themes#253]
- [x] [#286] Focus style should be applied when focus some cell vie keyboard navigation.
http://kendobuild-windows.openstack.progress.com/kendo-ui/treelist/keyboard-navigation - the user is not able to see which is the currently focused cell.
- [x] ~~Hover style is not removed after clicking on some button (http://kendobuild-windows.openstack.progress.com/kendo-ui/treelist/editing-incell  click 'delete' button for example)~~

**Comment by PreslavKozovski** [comment]
The second issue is JavaScrip related. I logged a new issue in the Kendo repo:
https://github.com/telerik/kendo/issues/9030
